packages/core/src/audio/AudioManager.ts

```
@@ -3,48 +3,114 @@
     let context = AudioManager._context;
     if (!context) {
-      AudioManager._context = context = new window.AudioContext();
+      try {
+        if (window.AudioContext) {
+          AudioManager._context = context = new window.AudioContext();
+          AudioManager._isWebKit = false;
+        } else if (window.webkitAudioContext) {
+          AudioManager._context = context = new window.webkitAudioContext();
+          AudioManager._isWebKit = true;
+        } else {
+          console.error('当前环境不支持AudioContext或webkitAudioContext');
```

ESLint

[error] Replace `'当前环境不支持AudioContext或webkitAudioContext'` with `"当前环境不支持AudioContext或webkitAudioContext"`

(prettier/prettier)